datahub-web-react/src/app/entity/shared/embed/UpstreamHealth/utils.ts

```diff
@@ -61,3 +11,37 @@ export function getNumAssertionsFailing(dataset: Dataset) {
 
     return numFailing;
 }
+
+export const DATASET_COUNT = 5;
+
+export function generateQueryVariables(
+    urn: string,
+    filterField: string,
+    start: number,
+    includeAssertions: boolean,
+    skip?: boolean,
+    count?: number,
+) {
+    return {
+        skip,
+        variables: {
+            input: {
+                urn,
+                query: '*',
+                types: [EntityType.Dataset],
+                start,
+                count: count !== undefined ? count : DATASET_COUNT,
+                direction: LineageDirection.Upstream,
+                orFilters: [
+                    {
+                        and: [
+                            { field: 'degree', condition: FilterOperator.Equal, values: ['1', '2', '3+'] },
+                            { field: filterField, condition: FilterOperator.Equal, values: ['true'] },
+                        ],
+                    },
+                ],
+            },
+            includeAssertions,
```

Comment:
do we need to change anything in our grphql file to support this $includeAssertions parameter?

Reply:
Great point, I'll double check. I remember being surprised I didn't see server-side usage of this but I maybe was on the wrong branch when I grep'd.

Reply:
I'm only seeing this usage on acryl-main: https://github.com/acryldata/datahub-fork/blob/acryl-main/datahub-web-react/src/graphql/search.graphql#L824